docs/release-notes/8.0.34-26.md

```diff
@@ -0,0 +1,150 @@
+
+  * [PS-5945](https://jira.percona.com/browse/PS-5945): Adds the MySQL server host name to `audit_log` details when writing to the SYSLOG.
+
+* [PS-8848](https://jira.percona.com/browse/PS-8848): Increases verbosity of the connection control plugin.
+
+## Improvements
+
+*  [PS-8728](https://jira.percona.com/browse/PS-8728): Adds the data masking component to align Percona Data Masking functionality with Oracle 8.0.33.
+
+* [PS-8624](https://jira.percona.com/browse/PS-8624): Exposes the number of TABLE objects with triggers in the Table Cache.
+
+* [PS-8742](https://jira.percona.com/browse/PS-8742): Adds the `audit_log_filter_database` system variable.
+
+## Bug fixes
+
+* [PS-8811](https://jira.percona.com/browse/PS-8811): There were linking issues with protocol buffers and macOS 12 on Azure Pipelines.
+
+* [PS-3942](https://jira.percona.com/browse/PS-3942): The MySQL ping command caused the `command_class="error"` in the audit log output.
+
+* [PS-8504](https://jira.percona.com/browse/PS-8504): Setting a large value of `audit_log_filter_buffer_size` caused the server exit. I 
+
+* [PS-8770](https://jira.percona.com/browse/PS-8770): The `INSTANT DDL` caused Percona Server 8.0.32 exit after upgrading from 5.7.41.
+
+* [PS-8772](https://jira.percona.com/browse/PS-8772): Stall of instance after INSERT (...) SELECT
+
+* [PS-8785](https://jira.percona.com/browse/PS-8785): The instance was completely stalled when using `INSERT (...) SELECT`.
+
+* [PS-8797](https://jira.percona.com/browse/PS-8797): The audit log plugin installation caused the server exit.
+
+* [PS-8810](https://jira.percona.com/browse/PS-8810): The `FEDERATED` update on table could fail without notifications when the table contained `DOUBLE` columns.
+
+* [PS-8859](https://jira.percona.com/browse/PS-8859): The `audit_log_filter` build failed on gcc-8.
+
+* [PS-8869](https://jira.percona.com/browse/PS-8869): The semi-sync master waited for ack when semi-sync replica was down and when was replaced with async.
+
+* [PS-8871](https://jira.percona.com/browse/PS-8871): The RESOURCE GROUP hint failed when it was used within a prepared statement.
+
+* [PS-8877](https://jira.percona.com/browse/PS-8877): The server exited when selecting from `performance_schema.innodb_redo_log_files`.
+
+The following bugs were fixed by imlementing the Audit Log Filter plugin:
+
+* [PS-5784](https://jira.percona.com/browse/PS-5784): The default `audit_log_format` parameter value for MySQL 8 was NEW but the default value for the Percona Server for MySQL Audit log plugin was OLD. 
+
+* [PS-6863](https://jira.percona.com/browse/PS-6863): The `audit_log` plugin did not handle the application messages from the `audit_api_message_emit` component.
+
+* [PS-8592](https://jira.percona.com/browse/PS-8592): XCom connection stalled forever in the `read()` syscall over the network. This issue prevented nodes from re-joining the cluster.
```

Comment:
This one is not Audit Log-related. Probably, move above the `The following bugs were fixed by imlementing the Audit Log Filter plugin` section.

Reply:
Moved